### PR TITLE
getting started with web3 doc

### PR DESCRIPTION
**Pull request by gzhao408**

### CODE REVIEW

File: src/web3-intro.md [hunk 1]
```diff
@@ -43,15 +43,21 @@ const URL = 'http://localhost:8545';
 const provider = new HDWalletProvider(MNEMONIC, URL);
 ```
 
-To connect to the Oasis devnet, use the link `'wss://web3.devnet.oasiscloud.io/ws'`. 
-You will likely want to connect your own account, i.e. the one you opened through the [Oasis Dashboard](https://dashboard.oasiscloud.io).
-You can get your seed phrase like this.
+**Connecting to Oasis Devnet:** To connect to the Oasis devnet, use the link `'wss://web3.devnet.oasiscloud.io/ws'`.
```
Comment (nhynes) [inline src/web3-intro.md]:
```suggestion
**Connecting to Oasis Devnet:** To connect to the Oasis Devnet, use the link `wss://web3.devnet.oasiscloud.io/ws`. 
```

File: src/web3-intro.md [hunk 2]
```diff
@@ -43,15 +43,21 @@ const URL = 'http://localhost:8545';
 const provider = new HDWalletProvider(MNEMONIC, URL);
 ```
 
-To connect to the Oasis devnet, use the link `'wss://web3.devnet.oasiscloud.io/ws'`. 
-You will likely want to connect your own account, i.e. the one you opened through the [Oasis Dashboard](https://dashboard.oasiscloud.io).
-You can get your seed phrase like this.
+**Connecting to Oasis Devnet:** To connect to the Oasis devnet, use the link `'wss://web3.devnet.oasiscloud.io/ws'`. 
+You will likely want to connect your own account; do this by providing your mnemonic (if you're using an HD wallet), private key, or array of private keys.
+Unforunately, the [Oasis Dashboard](https://dashboard.oasiscloud.io) accounts are not compatible with Web3.
```
Comment (nhynes) [inline src/web3-intro.md]: You can remove the "Unforunately" (sic).

File: src/web3-intro.md [hunk 3]
```diff
@@ -32,8 +33,7 @@ const Web3 = require('web3');
 const HDWalletProvider = require('truffle-hdwallet-provider');
 const Tx = require('ethereumjs-tx').Transaction;
 ```
-
-Initializing the HD Wallet provider requires a seed phrase (or mnemonic) from which it extracts your accounts (private keys and public addresses). 
+**Connecting to Local Oasis Chain:** Initializing the HD Wallet provider requires a seed phrase (or mnemonic) from which it extracts your accounts (private keys and public addresses). 
 It also requires a URL to create a connection through which all communication to the network will be done. 
 To connect to your local `oasis chain`, use `'http://localhost:8545'` and the menmonic provided to you there.
```
Comment (nhynes) [inline src/web3-intro.md]: `mnemonic`

File: src/web3-intro.md [hunk 4]
```diff
@@ -112,7 +114,7 @@ let provider = new HDWalletProvider(private_keys, URL, 0, num_keys);
 
 ## Getting Funded
 
-Unfortunately, our faucet is now deprecated. 
+Our faucet is now deprecated.
```
Comment (nhynes) [inline src/web3-intro.md]: good catch

File: src/web3-intro.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+- [Intro to Web3](#intro-to-web3)
+- [Connecting Wallets](#connecting-wallets)
+- [Getting Funded](#getting-funded)
+- [Connecting to Oasis.js](#connecting-to-oasis.js)
+
+## Intro to Web3
+
+[Web3.js](https://web3js.readthedocs.io/en/v1.2.0/web3-eth.html) is a Javascript module used by many Ethereum developers to deploy and interact with deployed smart contracts on various Ethereum networks. 
+The Oasis network fully supports Web3 interactions, so you can use it to deploy your contracts (read the docs for deploying [Solidity](./tutorials/deploy-solidity.md) and [Vyper](./tutorials/deploy-vyper.md) contracts) and interact with them.
```
Comment (eauge) [inline src/web3-intro.md]: I would add that oasis supports web3 **but** web3 cannot exploit the privacy and confidentiality preserving guarantees of the oasis platform. For that, the user can use our rust library

File: src/web3-intro.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+## Getting Funded
+
+Our faucet is now deprecated. 
+The best way to get funded is to email <support@oasislabs.com> with your public address. 
+
+## Connecting to Oasis.js
+
+[`oasis.js`](https://oasis-labs-oasis-client.readthedocs-hosted.com/en/latest/index.html) is our version of Web3, a Javascript SDK for building applications on top of Oasis platform services. 
+In `oasis.js`, you use a `wallet` and a `gateway`, similarly to how you needed a `MNEMONIC` and `URL` in Web3. 
+Once you've deployed contracts or sent transactions on an Oasis network via `Web3` or `oasis.js`, everything is accessible from either endpoint.
```
Comment (eauge) [inline src/web3-intro.md]: can you add a tutorial link to using oasis.js?

**Pull request merged**